RH_27_Make_Endpoint_for_FAQ_and_FAQ_table

antonVivoda opened the pull request:
Created FAQ table and Endpoint for FAQ

begjosip commented:
In faq schema, question and answer can be set to text instead of varchar 255 due to faster fetching and will solve longer questions and answers problem. Format all classes with ctrl alt l. In GetMapping remove ADMIN in logging. And in PostMapping don't map to Faq direct. Create class FaqRequest, then in service from request create Faq.

antonVivoda commented:
In faq schema question and answer type changed to text, all classes have been formatted, removed ADMIN in GetMapping logging, created FaqRequest and updated PostMapping

antonVivoda commented:
added @validated in PostMapping function, removed id from FaqRequest, removed setting id in FaqService class
